### DISCUSSION TITLE
Here are some awesome Azure AI Samples ... any others you would like to see here?

### DISCUSSION THREAD

**Discussion by achandmsft** [Azure-Samples/azure-ai#2] [General]
I have created an initial list of [Azure AI Code Samples](https://github.com/Azure-Samples/azure-ai/blob/main/README.md#azure-ai-code-samples) that developers find useful in the readme. 

If you know of other Azure AI SDK samples in GitHub that you find useful, let me know. What other samples would you like to see?

**Comment by ajmal-yazdani** [comment]
Hi @achandmsft, just started the exploration. Thanks for sharing all the doc/poc/sample.

I would like to know if there is any sample where I can talk with relational (Postgres, SQL) and non-relation database (Influx, Mongo)?

Please share!
